jetty-http/src/main/java/org/eclipse/jetty/http/SpecComplianceListener.java

```diff
@@ -0,0 +1,62 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2018 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.http;
+
+/**
+ * Listener for Spec Compliance Violations
+ *
+ * @see HttpCompliance
+ * @see CookieCompliance
+ * @see MultiPartFormDataCompliance
+ */
+public interface SpecComplianceListener
+{
+    /**
+     * The reference to a specific place in the Spec
+     */
+    interface SpecReference
```

Comment:
and maybe just `Reference` will do as `ComplianceListener.Reference` is sufficient to tell us what it is